dockstore-webservice/src/main/java/io/swagger/api/impl/ToolsImplCommon.java

```diff
@@ -223,6 +221,8 @@ public static Tool convertEntryToTool(Entry<?, ?> container, DockstoreWebservice
                 if (!descriptorType.isEmpty()) {
                     EnumSet<DescriptorType> set = EnumSet.copyOf(descriptorType);
                     toolVersion.setDescriptorType(Lists.newArrayList(set));
+                    // can assume in Dockstore that a single version only has one language
+                    toolVersion.setDescriptorTypeVersion(Map.of(set.stream().findFirst().get().toString(), Lists.newArrayList(version.getDescriptorTypeVersions())));
```

Comment:
Instead of `set.stream().findFirst().get().toString()`, I think you could just have `descriptorType.toString()`, which to me is more legible.